src/Nest/ElasticClient-Reindex.cs

```diff
@@ -14,5 +14,11 @@ public IObservable<IReindexResponse<T>> Reindex<T>(Func<ReindexDescriptor<T>, Re
 			var observable = new ReindexObservable<T>(this, _connectionSettings, reindexDescriptor);
 			return observable;
 		}
+
+		/// <inheritdoc />
+		public IObservable<IReindexResponse<object>> Reindex(Func<ReindexDescriptor<object>, ReindexDescriptor<object>> reindexSelector)
```

Comment:
Maybe we can default the type of the non generic to return `IReindexResponse<IDocument>` the lazy deserialization of `IDocument` is slightly more useful then `object` in this case.